releases/V2.1.0.md

```diff
@@ -0,0 +1,687 @@
+[1077 - [BUG] [OPS] L0p execution OL_CR[0] execution without output](https://github.com/COPRS/rs-issues/issues/1077)  
+[1076 - [BUG] [S3 PUG NRT] Browse image generation blocked by unavailable AUX files](https://github.com/COPRS/rs-issues/issues/1076)  
+[1067 - [BUG] SCDF not available due to skipper crash](https://github.com/COPRS/rs-issues/issues/1067)  
+[1065 - [BUG] Index settings incompatible with long coordinates](https://github.com/COPRS/rs-issues/issues/1065)  
+[944 - [BUG] [S3 SL2] Task table is incorrect when several SL1 products are available](https://github.com/COPRS/rs-issues/issues/944)  
+[1063 - [BUG] Invalid geojson shape in prip product](https://github.com/COPRS/rs-issues/issues/1063)  
+[969 - [BUG] RS native API and RS-API - CORS request did not succeed](https://github.com/COPRS/rs-issues/issues/969)  
+[1053 - [BUG] [S3] Around 50% of NTC  "SR_1_SRA_A_"  product have got a size of 2970 seconds (instead of 3029 seconds).](https://github.com/COPRS/rs-issues/issues/1053)  
+[1027 - [BUG] Metadata extraction failed for some products S3B_SR_1_LAN_RD_ and S3B_SR_1_SRA_A .](https://github.com/COPRS/rs-issues/issues/1027)  
+[1033 - [BUG] S3-PUG-STC IPF crashes on some S3A_SY_2_SYN products](https://github.com/COPRS/rs-issues/issues/1033)  
+[1028 - [BUG] [Distribution v1.13.1-rc1+WA982] Fails to extract  OL1 / OL2 products generated by PUG](https://github.com/COPRS/rs-issues/issues/1028)  
+[1029 - [BUG] SR1 1.14.0 - Some jobs failed with the error "CFI Orbit interpolation failed."](https://github.com/COPRS/rs-issues/issues/1029)  
+[709 - [BUG] Almost one trace is stored twice on the Elastic Search index processing_trace* .](https://github.com/COPRS/rs-issues/issues/709)  
+[822 - [BUG] Metrics "rs_pending_processing_job" are not available for chains S1 and S3.](https://github.com/COPRS/rs-issues/issues/822)  
+[1037 - [BUG] [OPS] Some compression task interrupted due to pod's end of life stuck the compression for this product.](https://github.com/COPRS/rs-issues/issues/1037)  
+[378 - [BUG][GRAFANA] No backup for library panel ](https://github.com/COPRS/rs-issues/issues/378)  
+[991 - [BUG] UWC - "ProductType+contains" queries returns no result if uppercase](https://github.com/COPRS/rs-issues/issues/991)  
+[961 - [BUG] [OPS] Impossible to use safescale cluster node delete](https://github.com/COPRS/rs-issues/issues/961)  
+[871 - [BUG] End-points "native-api", "scdf', "thanos" is not reachable from Black Box Exporter.](https://github.com/COPRS/rs-issues/issues/871)  
+[615 - [BUG] compression-part1-message-filter loops on error ErrorCallbackNotImplemented.](https://github.com/COPRS/rs-issues/issues/615)  
+[816 - [BUG] Missing S3-SM2-HY/LI/SI images on artifactory](https://github.com/COPRS/rs-issues/issues/816)  
+[927 - [BUG] [S2] IPF error : "No Antimerian segment found in two-parts Geo footprint"](https://github.com/COPRS/rs-issues/issues/927)  
+
+## Advice for use of the software configuration item
+  
+NC
```

Comment:
Non Compliant ??? I would expect "N/A -> Non Applicable"

Reply:
I didn't change this one. It was in the original document, I just copied/pasted this line, but I change it in the new version.